test/integration/testutils/utils.go

```diff
@@ -33,6 +35,33 @@ func FileExists(filePath string) (bool, error) {
 	return false, err
 }
 
+func DisableAutoTfVarsFile(rootDir string) error {
```

Comment:
Curious, why is this now required?  Shouldn't the `*.example.tfvars` be ignored anyway?

Reply:
Also because of the added symlinks on 3-fleetscope